flutter/lib/src/event_processor/flutter_enricher_event_processor.dart

```diff
@@ -155,7 +155,7 @@ class FlutterEnricherEventProcessor extends EventProcessor {
       // Also always fails in tests.
       // See https://github.com/flutter/flutter/issues/83919
       // 'window_is_visible': _window.viewConfiguration.visible,
-      'renderer': getRendererAsString()
+      'renderer': _options.rendererWrapper.getRendererAsString()
```

Comment:
Let's pass the `rendererWrapper` to `FlutterEnricherEventProcessor` via ctor so we dont need to add to `FlutterEnricherEventProcessor`

Reply:
It's using options in other places as well.